pipeline_dp/dp_engine.py

```diff
@@ -146,7 +147,7 @@ def _select_private_partitions(self, col, max_partitions_contributed: int):
             collection of elements (partition_key, accumulator)
         """
         budget = self._budget_accountant.request_budget(
-            noise_kind=NoiseKind.GAUSSIAN)
+            mechanism_type=MechanismType.GAUSSIAN)
```

Comment:
Please change to MechanismType.GENERIC

**Explanation:** Here it's used truncated geometric partition strategy, which nor Laplace, nor Gaussian. It's one of the motivation why we need Generic mechanism

Reply:
Makes sense, thank you! Done 👍